src/containers/my-quizzes/create-or-edit-quiz-container/CreateOrEditQuizContainer.tsx

```diff
@@ -198,6 +199,28 @@ const CreateOrEditQuizContainer = ({
         })
       : void addNewQuiz({ title, description, items: questions, category })
 
+  const CreateQuestionButton = isCreationOpen ? (
```

Comment:
You don't need to repeat the button. Instead, you can write a condition inside the `title`:
```js
<Tooltip
  ...
  title={
    isCreationOpen ? t('myResourcesPage.quizzes.savePreviousQuestion') : ''
  }
>
```
Then, according to the [documentation](https://mui.com/material-ui/api/tooltip/#props), the Tooltip will be displayed only if a title present: "Zero-length titles string, undefined, null and false are never displayed".

Reply:
Fixed! Thanks for your help)